CMakeLists.txt

```diff
@@ -75,6 +76,7 @@ set(WAMR_ROOT_DIR ${CMAKE_CURRENT_LIST_DIR}/third-party/wamr)
 set(WAMR_SHARED_DIR ${WAMR_ROOT_DIR}/core/shared)
 set(WAMR_IWASM_DIR ${WAMR_ROOT_DIR}/core/iwasm)
 set(WAMR_BUILD_TARGET X86_64)
+set(WAMR_EXECUTION_MODE AoT) # Select the WAMR execution Mode (AoT/Interpreter)
```

Comment:
This is great, but should also be set as a project option at the top for the same reasons as `FAASM_SGX_SUPPORT`. As there are only two options, I'd set it as:

```
option(WAMR_INTERPRETER_MODE "Turns interpreter on/ off (vs. AoT)" OFF)
```

This will simplify the `if` statement below.